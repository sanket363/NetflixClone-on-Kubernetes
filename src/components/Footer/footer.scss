@import '../../static/sass/style';

.footer {
  background-color: $color-background;
  padding-top: 15rem;
  padding-bottom: 4rem;
  grid-column: 1 / 13;
  text-align: center;

  @include responsive(phone) {
    padding-top: 10rem;
  }

  &__copyright {
    color: #fff;
    font-size: 1.8rem;

    &--link {
      text-decoration: none;
      color: $color-red-2;
      transition: all 0.3s;
    }

    &--link:hover {
      color: orange;
    }
  }
}
