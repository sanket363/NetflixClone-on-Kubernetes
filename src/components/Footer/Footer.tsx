import React from 'react'
import './footer.scss'

const footer = () => (
  <footer className='footer'>
    <div className='footer__copyright'>
      &copy; {new Date().getFullYear()} Made with ❤️ by{' '}
      <a
        className='footer__copyright--link'
        href='https://github.com/devandres-tech'
      >
        Dev Andres
      </a>
    </div>
  </footer>
)

export default footer
