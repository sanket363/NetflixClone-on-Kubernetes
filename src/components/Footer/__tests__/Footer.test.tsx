import { render } from '@testing-library/react'
import { FooterDefault } from './Footer.stories'

describe('Footer constructor', () => {
  test('should render default footer', () => {})
  const { container } = render(<FooterDefault />)
  expect(container).toMatchSnapshot()
})
