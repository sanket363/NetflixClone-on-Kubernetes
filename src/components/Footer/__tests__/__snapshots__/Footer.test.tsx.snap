// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="footer__copyright"
    >
      © 
      2022
       Made with ❤️ by
       
      <a
        class="footer__copyright--link"
        href="https://github.com/devandres-tech"
      >
        Dev Andres
      </a>
    </div>
  </footer>
</div>
`;
